apps/charterafrica/src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,84 @@
+import RichTypography from "@/commons-ui/core/RichTypography";
+import Section from "@/commons-ui/core/Section";
+import { Link } from "@commons-ui/next";
+import { Button } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { secondary, neutral } from "@/charterafrica/colors";
+import BackgroundBox from "@/charterafrica/components/BackgroundBox";
+import RichText from "@/charterafrica/components/RichText";
+
+const ErrorPage = React.forwardRef(function Error(props, ref) {
```

Comment:
This doesn't look like a Page to be... It should probably just be named `Error` i.e. just another component.